training/train.py

```diff
@@ -309,24 +324,66 @@ def exec_command(command_args, max_wait_time):
         raise s
 
 
-def create_git_pr(s3_client, model_version, recall_at_30):  # pragma: no cover
+def get_deployment_yaml():
```

Comment:
As you are writing new functions, why dont you write them in a new file rather than adding it here.

Reply:
We will not need such function to edit yaml file and raise SAAS PR. In new approach I am planing to use some python packages that can do heavy lifting for us to read and edit the yaml files and raise PR instead of customer code with bash script. This change was just to patch the current code to fix logical issue on model selection.

Reply:
I am hoping that this will be refactored.